WtWSMS/history/characters/old_frankish.txt

```diff
@@ -129,7 +129,7 @@
 }
 
 42 = {
-	name = "Lanthilde"	#PLRE vol. 2 Lantechildis
+	name = "Lanthilda"	#PLRE vol. 2 Lantechildis
```

Comment:
Shouldn't this be Lantechildis in that case?

Reply:
Well, going by the `name_list_old_frankish` the -is suffix is used for feminine names.

Reply:
I'll revert it for this PR and am going to discuss it with LM (as he's looking into improving name lists anyway)